Workbooks/AzureLogCoverage.json

```diff
@@ -0,0 +1,1761 @@
+                          "text": ""
+                        },
+                        {
+                          "operator": "Default",
+                          "thresholdValue": null,
+                          "representation": "Blank",
+                          "text": "{0}{1}"
+                        }
+                      ]
+                    }
+                  },
+                  {
+                    "columnMatch": "LAW Connection Status",
+                    "formatter": 5
+                  },
+                  {
+                    "columnMatch": "Enable",
+                    "formatter": 7,
+                    "formatOptions": {
+                      "linkTarget": "Resource",
+                      "subTarget": "Diagnostics",
+                      "linkLabel": "Open Audit Log Settings",
+                      "linkIsContextBlade": true,
+                      "bladeOpenContext": {
+                        "bladeName": "DiagnosticsLogsBlade",
+                        "extensionName": "Microsoft_Azure_Monitoring",
+                        "bladeJsonParameters": "{\r\n    \"id\": \"{id}\"\r\n}\r\n"
+                      }
+                    }
+                  }
+                ],
+                "hierarchySettings": {
+                  "treeType": 1,
+                  "groupBy": [
+                    "LAW Connection Status"
+                  ],
+                  "finalBy": "subscriptionId"
+                },
+                "labelSettings": [
+                  {
+                    "columnId": "subscriptionId",
+                    "label": "Subscription"
+                  },
+                  {
+                    "columnId": "resourceGroup",
+                    "label": "Resource Group"
+                  },
+                  {
+                    "columnId": "id",
+                    "label": "Resource"
+                  },
+                  {
+                    "columnId": "sentinelEnabled",
+                    "label": "Sentinel Enabled"
+                  },
+                  {
+                    "columnId": "sku",
+                    "label": "Sku"
+                  },
+                  {
+                    "columnId": "dataIngestionStatus",
+                    "label": "Data Ingestion Status"
+                  },
+                  {
+                    "columnId": "dailyQuotaGb",
+                    "label": "Daily Quota in GB/day"
+                  },
+                  {
+                    "columnId": "retentionInDays",
+                    "label": "Retention in Days"
+                  }
+                ]
+              }
+            },
+            "name": "queryLAWMerge"
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "parameterTab",
+        "comparison": "isEqualTo",
+        "value": "law"
+      },
+      "name": "groupLAW"
+    }
+  ],
+  "fromTemplateId": "sentinel-UserWorkbook",
```

Comment:
Hello @anders-alex please change the templateid from "sentinel-UserWorkbook" to any other id.

 Please be consistent with the format ' sentinel-"workbookName" '

Reply:
> Hello @anders-alex please change the templateid from "sentinel-UserWorkbook" to any other id. There is already one workbook with the same templateID
> 
> Please be consistent with the format ' sentinel-"workbookName" '

still not resolved